pillar.example

```diff
@@ -26,6 +27,32 @@ alcali:
     db_port: 3306
     master_minion_id: master
     secret_key: 'thisisnotagoodsecret.orisit?'
-    allowed_hosts: '*'
     salt_url: 'https://master:8080'
     salt_auth: alcali
+
+    # auth_backend: ldap
```

Comment:
If auth_backend is not defined, then the state will fail to compile:

`    Rendering SLS 'base:alcali.package.install' failed: Jinja variable 'dict object' has no attribute 'auth_backend'`